ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -342,6 +349,8 @@ function build_upi_cluster() {
 
 trap 'error_handler $? $LINENO' ERR
 
+echo "Started creation of PowerVS cluster"
+
```

Comment:
```suggestion
```
There should already be comments in the functions at the beginning of the methods

Reply:
It was used for the starting point , yes you are this was not required . I will remove it .